src/common/runtime/helper/options.ts

```diff
@@ -100,6 +102,7 @@ export const kCTSOptionsInfo: OptionsInfos<CTSOptions> = {
   debug: { description: 'show more info' },
   compatibility: { description: 'run in compatibility mode' },
   forceFallbackAdapter: { description: 'pass forceFallbackAdapter: true to requestAdapter' },
+  enforceDefaultLimits: { description: 'force the adapter limits to the default limits' },
```

Comment:
> So, I made it so it makes a temp device. This might be fine but, tests that test `low-power` vs `high-performance` may have issues.

That seems fine since I don't think this option isn't necessarily meant to work with every test. Perhaps make a note of that here though.

Reply:
added